Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -3,20 +3,19 @@ Copyright (c) 2019 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Yaël Dillies
 -/
-import Mathlib.Algebra.Module.BigOperators
-import Mathlib.Data.Finset.NoncommProd
+
```

Comment:
```suggestion
import Mathlib.Algebra.Module.BigOperators
```

Reply:
I don't understand what will be the effect of this suggestion.

Reply:
I am ordering alphabetically the imports